newbank/server/NewBank.java

```diff
@@ -177,6 +177,7 @@ private String paybackLoan(CustomerID customerID, int loanNumber) {
                 amount += loan.getAmount() / 100 * loan.getInterest();
                 accountFrom.debit(amount);
```

Comment:
The balance of the account should be tested for sufficient funds before a transfer is made.

Reply:
Thank you, I will add that.

Reply:
Nicely done

Reply:
The account balance is now checked for sufficient funds.